frontend/packages/erd-core/src/features/erd/components/ERDRenderer/CommandPalette/CommandPalette.tsx

```diff
@@ -22,8 +22,6 @@ export const CommandPalette: FC = () => {
   const table = schema.current.tables[tableName ?? '']
   const { selectTable } = useTableSelection()
 
-  const [focusedTableName, setFocusedTableName] = useState<string | null>(null)
```

Comment:
I omitted this feature to make the code simpler for now 😢 
I think implement logics for click and double click, with/without holding ⌘ key, involves too much patterns and will be very complex.
https://github.com/liam-hq/liam/pull/2127